articles/guides/login/migrating-lock-v10-webapp.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Moving Web Applications using Lock to Centralized Login 
+description: Learn how to migrate from Web Applications using Lock to Centralized Login
+toc: true
+---
+# Migrate Web Applications using Lock 10+ to Centralized Login
+
+This document explains how to migrate Web Applications using Lock to centralized login. 
+
+For other migration scenarios see [Migrating from Embedded to Centralized Login](/guides/login/migration-embedded-centralized).
+
+When you use Lock in a Web Application, your code does basically this:
+
+1. Initialize Lock using `responseType = 'code'`:
+
+```js
+var lock = new Auth0Lock('${account.clientId}', '${account.namespace}', {
+     auth: {
+      redirectUrl: '${account.callback}',
+      responseType: 'code',
+      audience: 'https://' + '${account.namespace}' + '/userinfo',
+      params: {
+        scope: 'openid'
+      }
+    }
+}); 
+```
+2. Show lock when the login button is clicked:
```

Comment:
I'd change it to "Show Lock when a login is required" (or triggered), because it could be a login button but also the user trying to access a protected resource.